deploy/roles/neo4j/tasks/main.yml

```diff
@@ -0,0 +1,62 @@
+---
+
+- name: Add neo4j user
+  user:
+    name: "{{ neo4j_user }}"
+    system: yes 
+    home: "{{ neo4j_dir }}"
+    shell: "{{ neo4j_user_shell }}"
+    state: present
+    #groups: "{{ neo4j_user_groups | join(',') }} "
```

Comment:
@olivermel if this isn't needed, please remove the line rather than commenting it out.